channeld/channel_wire.csv

```diff
@@ -62,10 +63,10 @@ channel_init,,last_remote_secret,struct secret
 channel_init,,lflen,u16
 channel_init,,localfeatures,lflen*u8
 
-# master->channeld funding hit new depth >= lock depth
-channel_funding_locked,1002
-channel_funding_locked,,short_channel_id,struct short_channel_id
-channel_funding_locked,,depth,u32
+# master->channeld funding hit new depth(funding locked if >= lock depth)
+channel_funding_depth,1002
+channel_funding_depth,,short_channel_id,?struct short_channel_id
```

Comment:
Ok, I believe normally cdecker does the first review for new contributors, but I will give already some comments (hopefully I give better advice this time).

That is a nice trick to get around first call where channel->scid=NULL !

Reply:
Thank you Simon, I always benefit from your comments and advice!

Reply:
> Ok, I believe normally cdecker does the first review for new contributors, but I will give already some comments (hopefully I give better advice this time).
> 
> That is a nice trick to get around first call where channel->scid=NULL !

Thank you Simon, I always benefit from your comments and advice!